sycl/test/matrix/matrix-cuda.cpp

```diff
@@ -0,0 +1,153 @@
+// REQUIRES: gpu, cuda
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -Xsycl-target-backend --cuda-gpu-arch=sm_80 -DSYCL_EXT_ONEAPI_MATRIX=3  %s -o %t.out
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::intel::experimental::matrix;
```

Comment:
change to:
using namespace sycl::ext::oneapi::experimental::matrix;

Reply:
Thanks, done.